### ISSUE TITLE
Rollback to Harmony 1.2 or use shared Harmony 2.x dependency

### ISSUE THREAD

**Issue by boformer** [bernardd/Crossings#6]
We noticed from user bug reports that you are using Harmony 2.0.0.8.

This version of Harmony is not compatible with widely used 1.2.0.1.

We have been working on a central Harmony 2.0.0.9 mod that restores compatibility. I would strongly recommend you to make use of that mod (there is a small utility that auto-subscribes it on game startup, so there is no disruption for existing users).

Alternatively you could roll back to Harmony version 1.2 which would make it compatible with other 1.2 mods (99%) and mods using 2.0.0.9 through the shared dependency.

See https://github.com/boformer/CitiesHarmony

Also see this reddit thread about Harmony 2.x drama: https://www.reddit.com/r/RimWorld/comments/fbnm45/harmony_the_full_story/ (it also partially applies to C:S).

I can help with the migration and submit a PR if you want.

**Comment by boformer** [comment]
Ah we noticed that you reverted to pre-Harmony versions shortly after you updated in march. well I guess then there is less pressure. with the shared Harmony mod you can could make the switch!

**Comment by bernardd** [comment]
Thanks - when I have a moment I'll reinstate the use of Harmony with the new system - it definitely makes the code an order of magnitude simpler, so here's hoping it goes more smoothly this time around.

**Comment by bernardd** [comment]
> I can help with the migration and submit a PR if you want.

@boformer I didn't actually notice this offer when you first posted, my apologies for that. The current `master` actually has my Harmony implementation - I just rolled back the version that had been released on Steam. If you're able to submit a PR using the harmony mod that would be awesome, thanks!

> Alternatively you could roll back to Harmony version 1.2

Unfortunately that one isn't an option - I hit this bug: https://github.com/pardeike/Harmony/issues/227 which is only fixed in v2.
